### DISCUSSION TITLE
Security: vulnerability found in keycloak quarkus distribution

### DISCUSSION THREAD

**Discussion by ahrycej** [keycloak/keycloak#15544] [Archive]
/opt/keycloak/lib/lib/main/com.squareup.okhttp3.okhttp-3.14.9.jar

the vulnerable jar file of square/okhttp is found in quarkus distribution from version 18.0.0 till 20.0.1

details on vulnerability:
https://github.com/square/okhttp/issues/6738

Could somebody assess the risk and update? The fix shall be available in 4.9

**Comment by cgeorgilakis** [comment]
Where does this severe secure vulnerability affect Keycloak?

Moreover, I can not understand how [this github issue](https://github.com/keycloak/keycloak/issues/14641) may solve the issue. Okhttp version does not go to 4.9.2

**Comment by ahrycej** [comment]
Could somebody answer to this ticket, the issue was not removed from the keycloak 20.0.0 or it is not clear what was the fix since version of okhttp was not updated. Thank you

**Comment by NicolasLiampotis** [comment]
Is there any update on this?

**Reply by ahrycej** [reply]
I have the most recent keycloak version and this is still present.

**Reply by ahrycej** [reply]
here you can see all issues I identified in keyckloak:
https://github.com/keycloak/keycloak/issues/16075
